src/commons/services/utils.js

```diff
@@ -222,6 +222,62 @@
             return poolList;
         };
 
+        vm.getRBDsDetails = function(clusterId) {
+            var rbdList = [], index1, index2;
+
+            if($rootScope.clusterData !== null) {
+                clusterData = $rootScope.clusterData.clusters;
+                len = clusterData.length;
+
+                for ( i = 0; i < len; i++ ) {
```

Comment:
@kamleshverma1 yes I totally understand that. But we can still divide them into functions. Lets keep it this way for now. Will refactor it after Monday.